CONTRIBUTING

```diff
@@ -19,16 +19,14 @@ more.
 
 The project maintains the following source code repositories
 
-* http://git.eclipse.org/c/pde/eclipse.pde.ui.git
-* http://git.eclipse.org/c/pde/eclipse.pde.build.git
+* https://github.com/eclipse-pde/eclipse.pde/
 
-This project uses Bugzilla to track ongoing development and issues.
+This project uses Github issues and pull requests to track ongoing development and issues.
 
-* Search for issues: https://eclipse.org/bugs/buglist.cgi?product=PDE
-* Create a new report: https://eclipse.org/bugs/enter_bug.cgi?product=PDE
+* Issues: https://github.com/eclipse-pde/eclipse.pde/issues
```

Comment:
Maybe it is also worth to mention the legacy bugzilla as such since it still provides a huge collection of past and also open issues.

Reply:
Bugzilla is IMHO so I will not refer to it. :-)